meta/main.yml

```diff
@@ -0,0 +1,54 @@
+galaxy_info:
+  author: Wolfgang Hotwagner
+  description: This role installs ghostsagent on Linux
+  company: Austrian Institute of Technology
+
+  # If the issue tracker for your role is not on github, uncomment the
+  # next line and provide a value
+  # issue_tracker_url: http://example.com/issue/tracker
+
+  # Choose a valid license ID from https://spdx.org - some suggested licenses:
+  # - BSD-3-Clause (default)
+  # - MIT
+  # - GPL-2.0-or-later
+  # - GPL-3.0-only
+  # - Apache-2.0
+  # - CC-BY-4.0
+  license: GPL-3.0-only
```

Comment:
I think we should check the license of ghosts too, since we are shipping the zip-files

Reply:
solved by downloading it during the installation